upgrades/3.x/3.16.0/README.adoc

```diff
@@ -0,0 +1,27 @@
+= Upgrade to 3.16.0
+
+== API definition import process changes
+
+Gravitee 3.16.0 introduces a new **crossId** in API definition, which identify entities across environments.
+This improves the API import and promotion processes reliability.
+
+If you are using the _API import from JSON definition_ feature,
+we **highly recommend** updating your API definitions by re-exporting your API.
+
+Otherwise, if you import an API definition which doesn't contain **crossId**,
+Gravitee will do the best-effort to import your API definition without it.
+
+Learn more about API definition import link:/apim/3.x/apim_publisherguide_import_apis.html[here].
```

Comment:
Not sure, this link will work. On top of that, we must ensure that links are relative.
```suggestion
Learn more about API definition import link:{{ '/apim/3.x/apim_publisherguide_import_apis.html' | relative_url}}[here].
```